From 770d417e28dc9527fec8b8a00caaba8825995454 Mon Sep 17 00:00:00 2001
From: Grace Chin <gchin@redhat.com>
Date: Wed, 19 Jul 2023 10:25:55 -0400
Subject: [PATCH] Fix: tools: Fix a bug in clone resource description display

Previously, descriptions of resources running on multiple
nodes were displayed despite --full not being used (with pcs
status) or --show-detail not being used (with crm_mon).

For example, clone resources running on multiple nodes were
affected.

Now, --full and --show-detail must be used in order for resource
descriptions to be displayed, regardless of the number of nodes
the resource is run on.

see bz: 2106642
---
 lib/pengine/pe_output.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/lib/pengine/pe_output.c b/lib/pengine/pe_output.c
index e0b43d997a..d1c9f6e226 100644
--- a/lib/pengine/pe_output.c
+++ b/lib/pengine/pe_output.c
@@ -20,8 +20,7 @@ pe__resource_description(const pe_resource_t *rsc, uint32_t show_opts)
 {
     const char * desc = NULL;
     // User-supplied description
-    if (pcmk_any_flags_set(show_opts, pcmk_show_rsc_only|pcmk_show_description)
-        || pcmk__list_of_multiple(rsc->running_on)) {
+    if (pcmk_any_flags_set(show_opts, pcmk_show_rsc_only|pcmk_show_description)) {
         desc = crm_element_value(rsc->xml, XML_ATTR_DESC);
     }
     return desc;
